rpc/src/rpc_actor.rs

```diff
@@ -129,11 +145,69 @@ impl Actor for RpcServer {
     }
 }
 
+async fn warm_up_rpc_cache(
+    chain_id: &ChainId,
+    block: &BlockHeaderWithHash,
+    env: &RpcServiceEnvironment,
+) {
+    // Async calls
+    let get_block_metadata =
+        crate::services::base_services::get_block_metadata(&chain_id, &block.hash, env);
+    let get_block = crate::services::base_services::get_block(&chain_id, &block.hash, &env);
+    let get_block_operations_metadata =
+        crate::services::base_services::get_block_operations_metadata(
+            chain_id.clone(),
+            &block.hash,
+            env,
+        );
+    let get_block_operation_hashes = crate::services::base_services::get_block_operation_hashes(
+        chain_id.clone(),
+        &block.hash,
+        &env,
+    );
+    let get_block_header = crate::services::base_services::get_block_header(
+        chain_id.clone(),
+        block.hash.clone(),
+        &env.persistent_storage(),
+    );
+    let _ = futures::join!(
+        get_block_metadata,
+        get_block,
+        get_block_operations_metadata,
+        get_block_operation_hashes,
+        get_block_header,
+    );
+
+    // Sync calls
+    let _ = crate::services::base_services::get_additional_data(
```

Comment:
@tizoc 
Bruno, add this `base_services::get_additional_data`  at the begining, to be cached first,
because all other calls use this internally, so once this is cached, all other will be faster and get it from cache :)

Reply:
@tizoc 
and we should the the same for calls:

```
BlockStorage::new(persistent_storage).get_with_json_data(
``